bundlegen/core/bundle_processor.py

```diff
@@ -536,29 +537,174 @@ def _process_logging(self):
+
+    def _determine_sublibs(self):
+        """
+        Determine sublibs inside libs info.
+        Specific processing for libc and its sublibs like libresolv.
+        First it determines the actual libc lib by finding the one
+        with the most GLIBC* versions inside its "Version definition section".
+        It also looks for other libs that only contain GLIBC* entries on their
+        "Version definition section". These are then assumed to be "sublibraries"
+        of libc.so
+        """
+        if not self.platform_cfg.get('libs'):
+            return
+
+        libs = self.platform_cfg['libs']
+        max = 0
+        libc = None
+        sublibs = []
+        for lib in libs:
+            m1 = re.match(r"^/lib/lib\S+\.so\.\d+$", lib['name'])
+            m2 = re.match(r"^/lib/ld-linux\S+\.so\.\d+$", lib['name'])
+            if not m1 and not m2:
+                continue
+
+            cnt = 0
+            for apiversion in lib['apiversions']:
+                if (apiversion.startswith('GLIBC_')):
+                    cnt += 1
+            if cnt == 0 or (cnt != len(lib['apiversions'])):
+                continue
+
+            sublibs.append(lib)
+            if (cnt > max):
+                libc = lib
+                max = cnt
+
+        if (libc is None):
+            return
+
+        logger.debug(f"Found libc: {libc['name']}")
+        sublibs.remove(libc)
+        libc['sublibs'] = []
+        for sublib in sublibs:
+            libc['sublibs'].append(sublib['name'])
+            sublib['parentlib'] = libc['name']
+            sublib['apiversions'] = []
+
+    def _take_rootfs_lib(self, srclib, dstlib, api_info):
+        """ The lib version from image rootfs was choosen. Basically log here that it was choosen.
+            Also for any sublibs if present.
+        """
+        logger.debug(f"IMAGE version choosen: {dstlib}")
+        self.handled_libs.add(dstlib)
+        if api_info and api_info.get('sublibs'):
+            for sublib in api_info['sublibs']:
+                logger.debug(f"IMAGE version choosen: {sublib}")
+                self.handled_libs.add(sublib)
+
+    # ==========================================================================
+    def _mount_or_use_rootfs(self, srclib, dstlib):
+        """Determine to mount lib from host OR use the one inside bundle rootfs.
+           If lib exists in rootfs and apiversions info exists for it inside *_libs.json config
+           then we try to use that version of the lib that has most API versions defined inside.
+           If lib does not exist inside image rootfs then create mount bind.
+           If no API versions info exists then create mount bind.
+           If decision cannot be made because API versions disjunct, choose image rootfs version.
+
+        Args:
+            lib (string): lib name like /lib/libc.so.6
+            mount_if_not_on_rootfs (bool): if lib doesn't exist inside rootfs
```

Comment:
`mount_if_not_on_rootfs` this doesn't seem to be current parameter. Now there are only `srclib` and `dstlib`. Also line above there is `lib` not `srclib` stating that it is name not path of library. 
Probably need to update those comments to reflect current code.